SU2_CFD/include/numerics_structure.hpp

```diff
@@ -2071,34 +2071,64 @@ class CUpwLin_AdjTurb : public CNumerics {
 };
 
 /*!
- * \class CUpwSca_TurbSA
- * \brief Class for doing a scalar upwind solver for the Spalar-Allmaral turbulence model equations.
+ * \class CUpwSca_Abstract
```

Comment:
No, I haven't looked at that.  I've used the higher order upwind schemes in the mass conservation equations, but I haven't tried to apply them to the turbulence transport equations.